content/sdk/10.js/00.ethers/20.v6/00.getting-started.md

```diff
@@ -1,69 +1,134 @@
 ---
 title: "Getting Started"
-description: "Explore zkSync-ethers SDK version 6"
-tags: ["zksync", "sdks", "zkSync Era SDK", "ethereum", "era network", "javascript", "v6"]
+description: "Explore zkSync-era SDK version 6"
```

Comment:
Why is this renamed to zkSync-era? Isn't the package named zksync-ethers?

Reply:
Typo.. I have updated it